src/components/AddExisitingStudentsModal.tsx

```diff
@@ -10,31 +10,30 @@ import {
   ModalBody,
   ModalCloseButton,
 } from '@chakra-ui/react'
-import { IStudent, IStudentInStudentGroup, IStudentGroup, IStudentToAdd, IStudentFact } from 'interfacesAndTypes'
+import { IStudentToAdd, IStudentFact } from 'interfacesAndTypes'
 import StudentPreview from 'components/StudentPreview'
-import firebase from 'firebase'
 import { useFirestore } from 'reactfire'
+import {
+  useStudentGroup,
+  useStudentsInThisStudentGroup,
+  useStudents,
```

Comment:
https://lh3.googleusercontent.com/proxy/cZW6q3Zdez-Z0BpPmX1uwVevCog8Ncm9dEE8mfvowvJwgpn8x-e58tKNK51UmqlgkM6RWTFiZYik6o1oIb7BUHRTTLzjqw

Reply:
I can't make this link work

"403. That’s an error.

Your client does not have permission to get URL /proxy/cZW6q3Zdez-Z0BpPmX1uwVevCog8Ncm9dEE8mfvowvJwgpn8x-e58tKNK51UmqlgkM6RWTFiZYik6o1oIb7BUHRTTLzjqw from this server. (Client IP address: 174.85.32.129)
"

Reply:
it's just a link to this face:

ಠ_ಠ

Reply:
like as in, `useStudents` ಠ_ಠ

Reply:
like as in `useStudents` disapproveFace

Reply:
carry on

Reply:
Did you notice my interface names for those return values?  Like IUseStudentsInGroups?